### PR TITLE
Boundary Config Update

### PR DESCRIPTION
**Pull request by sachiyuan**
- Update Boundary config to make it more generalized and does not need a unique name for the same type of boundary (e.g Counties) between regions
- Boundary request takes workspace name from conditions.json region_geoserver_name value to make updating workspace names easier 
- Corrects issue where only Southern California used vector name 'vector_huc-10' while Sierra Nevada and Central California used 'vector_huc10'
- Assumes geoservers are updated to have 'vector_huc10' available for all regions 
- Assumes same set of boundaries will be used for each region (Confirmed there is no indication that this will ever not be the case)

### CODE REVIEW

File: src/planscape/config/boundary.json [hunk 1]
```diff
@@ -28,177 +22,38 @@
         {
             "boundary_name": "huc10",
             "display_name": "HUC-10",
-            "vector_name": "southern-california:vector_huc-10",
-            "region_name": "none",
+            "vector_name": ":vector_huc10",
```
Comment (runforyourlives) [inline src/planscape/config/boundary.json:25]: double checking here - did the vector file for huc-10/huc10 in geoserver get renamed?

Comment (sachiyuan) [reply src/planscape/config/boundary.json:25]: George just put in the change in dev-geo and after we test it out he'll make the change in geo (prod)

Comment (runforyourlives) [reply src/planscape/config/boundary.json:25]: confirmed that both huc-10 and huc10 are loaded on dev-geo, which will be the same for prod before the next prod push

File: src/planscape/config/boundary.json [hunk 2]
```diff
@@ -1,25 +1,19 @@
 {
     "__comment1": "TODO: replace filename for SoCal's HUC10 when it loses its dash.",
```
Comment (runforyourlives) [inline src/planscape/config/boundary.json:2]: you can remove this line now since you're addressing this in this CL.

**Review by runforyourlives — APPROVED** [review]
Please update the comments here that you're also fixing the huc-10 dash config issue and assuming that geoservers are updated.

**Pull request merged**